lua/telescope/builtin/internal.lua

```diff
@@ -952,18 +952,28 @@ end
 
 -- TODO: make filtering include the mapping and the action
 internal.keymaps = function(opts)
-  local modes = { "n", "i", "c" }
-  local keymaps_table = {}
+  opts.modes = utils.get_default(opts.modes, { "n", "i", "c", "x" })
+  opts.show_plug = utils.get_default(opts.show_plug, true)
```

Comment:
no more `utils.get_default`. Please use `vim.F.if_nil` i am about to deprecate `get_default` #1545